kasa/httpclient.py

```diff
@@ -28,12 +29,18 @@ def get_cookie_jar() -> aiohttp.CookieJar:
 class HttpClient:
     """HttpClient Class."""
 
+    # Time to wait between requests if getting client os errors
+    WAIT_TIME = 0.5
```

Comment:
```suggestion
    WAIT_BETWEEN_REQUESTS_ON_OSERROR = 0.5
```
Maybe something more descriptive like this? The comment itself could briefly explain why this is necessary.

Reply:
Done